Sources/SemanticAnalyzer/SemanticAnalyzer+Expressions.swift

```diff
@@ -162,6 +162,23 @@ extension SemanticAnalyzer {
                 functionDeclaration: passContext.functionDeclarationContext!.declaration))
           }
         }
+
+        if functionCall.isOuterExternalCallFunctionCall {
```

Comment:
Why do we only check the outer call?

Reply:
Because there might be other calls as children of the main function call in the AST, and the hyperparameter checks should not apply to them too (unless they're being called with 'call' as well)